polaris-migrator/src/migrate.ts

```diff
@@ -48,7 +48,6 @@ export async function migrate(
       extensions: 'tsx,ts,jsx,js',
       parser: 'tsx',
       verbose: 2,
-      runInBand: true,
```

Comment:
Note: [`runInBand: true`](https://github.com/facebook/jscodeshift/blob/2d0463ac653bd52cf100dc4599f006481aa2b6e2/src/Runner.js#L223) will only [use one worker](https://github.com/facebook/jscodeshift/blob/2d0463ac653bd52cf100dc4599f006481aa2b6e2/src/Runner.js#L257-L260) to run the migrations